Library/Homebrew/extend/os/linux/tap.rb

```diff
@@ -0,0 +1,9 @@
+class CoreTap < Tap
+  def default_remote
+    if ENV["HOMEBREW_FORCE_HOMEBREW_ORG"]
+      "https://github.com/Homebrew/homebrew-core".freeze
+    else
+      "https://github.com/Linuxbrew/homebrew-core".freeze
```

Comment:
Will this direct people to file Linux formulae issues there (if we don't already do that)?

Reply:
The user is directed to `OS::ISSUES_URL`, which on Linux is https://github.com/Linuxbrew/brew/wiki/Troubleshooting.
As far as I can tell, `CoreTap.instance.issues_url` is not used when `official_tap?` is true.